modules/ROOT/pages/to-describe-an-asset.adoc

```diff
@@ -4,98 +4,94 @@ include::_attributes.adoc[]
 endif::[]
 :imagesdir: ../assets/images/
 
-You can edit an asset's portal; change its name, icon, description, tags, categories, and custom fields; and add portal pages with content.
+You can edit an asset's portal, change its name, icon, description, tags, categories, and custom fields; and add portal pages with content.
 
 All versions of an asset have the same name, icon, description, and computed average rating.
 
 Each minor version of an asset has its own tags, categories, custom fields, and content pages, including an optional page of terms and conditions.
 
 In Exchange, open the asset list and select the asset. Exchange shows the asset portal.
 
-== Edit Asset Portal Name
+== Edit Asset Portal Information
 
-. Select the asset portal name or the pen icon after the name.
-. Enter the new name.
-. (Optional) To cancel the changes, select the cancel icon.
-. To save the new name, press the enter or return key on your keyboard, or select the check icon.
+You can change the asset icon, name, description, contact name, and contact email. 
 
-NOTE: Before the August 2019 release, the versions of an asset could have different names. After the August 2019 release, a new name applies to all versions of the asset.
+. Click the edit pen icon in the asset portal.
```

Comment:
We should try to show this icon rather than describe it in terms that might have different implications for different audiences.